llama_stack/providers/registry/safety.py

```diff
@@ -22,6 +22,7 @@ def available_providers() -> List[ProviderSpec]:
             provider_type="inline::prompt-guard",
             pip_packages=[
                 "transformers",
```

Comment:
though why do we need this line? The one you added means "install transformers, with additional dependencies for accelerate", so this line is probably redundant.

Reply:
Fixed :+1: